lib/asset_checker.rb

```diff
@@ -0,0 +1,39 @@
+class AssetChecker
+  def self.run(argv)
+    assets_file = 'app/assets/javascripts/assets.js.erb'
+    translations_file = 'app/assets/javascripts/i18n-strings.js.erb'
+    @asset_strings = load_included_strings(assets_file)
+    @translation_strings = load_included_strings(translations_file)
+    exit_code = (argv.map { |f| check_file(f) }).any? ? 1 : 0
+    exit exit_code # rubocop:disable Rails/Exit
```

Comment:
Another option, that would make this easier to test, would be to have this return a success/false and do the `exit` inside the `scripts/check-assets` file

(in that file)
```ruby
has_missing = AssetChecker.new(ARGV).has_missing_assets?
if has_missing
  exit 1
else
  exit 0
end
```